server/src/service/machinesSchema.ts

```diff
@@ -37,12 +38,13 @@ export const MachinesTypeDefs = gql`
   input MachineFamilyInput {
     name: String!
     description: String
-    trainingModule: Int!
+    trainingModule: TrainingModule!
   }
 
   input ReservationInput {    
-    studentId: Int!
-    machineId: Int!
+    userId: User!
```

Comment:
From the graphql docs: "Input types can't have fields that are other objects, only basic scalar types, list types, and other input types". So unless user and machine are inputs I don't think this works. I think it is fine to just use an int (primary key) here

Reply:
@pjw9963 Yeah I was confused about this and then checked your inputs and you seem to have used types in them too. So we need to change these as well then? https://github.com/rit-construct-makerspace/makerspace/blob/71dd193f21bfa733f94340075d677dbf9b794e78/server/src/service/trainingSchema.ts#L38

Reply:
that's an enum not a type so I think that's okay to use